### PR TITLE
feat(lib): support for angular v10

### PR DESCRIPTION
**Pull request by smileLilith**
Upgrade Angular to v10. No breaking changes

### CODE REVIEW

File: projects/ngx-scroll-to/package.json [hunk 1]
```diff
@@ -55,11 +55,9 @@
     "node": ">=8.0.0",
     "npm": ">=6.0.0"
   },
-  "dependencies": {
-    "tslib": "^2.0.0"
-  },
   "peerDependencies": {
     "@angular/common": "^8.0.0 || ^9.0.0 || ^10.0.0",
-    "@angular/core": "^8.0.0 || ^9.0.0 || ^10.0.0"
+    "@angular/core": "^8.0.0 || ^9.0.0 || ^10.0.0",
```
Comment (daniel-seitz) [inline projects/ngx-scroll-to/package.json:60]: what about v11?

**Comment by JrFernando** [comment]
Forecast to approve?

File: projects/ngx-scroll-to/package.json (hunk 1 above)
Comment (smileLilith) [reply projects/ngx-scroll-to/package.json:60]: I haven't tested with v11 or attempt to upgrade to it. Feel free to do it :)

**Comment by cosminbondane** [comment]
Any news about this?

File: projects/ngx-scroll-to/package.json (hunk 1 above)
Comment (daniel-seitz) [reply projects/ngx-scroll-to/package.json:60]: v11 works for us, v12 is out by the way, haven't testd..

**Comment by akaustav** [comment]
@Jonnyprof / @rafa-as / @squelix / @SmallhillCZ / @yanis-git / @ssmietana-newforma / @izmaylovdev / @zakhenry:
Trying to gain attention to this PR by mentioning contributors.

**Comment by SmallhillCZ** [comment]
I am not a contributor to this repo (maybe I fixed a typo somewhere sometime), but I surely would like to see this merged ;)

File: projects/ngx-scroll-to/package.json (hunk 1 above)
Comment (HarelM) [reply projects/ngx-scroll-to/package.json:60]: v12 also works. I suggest to simply add >= 8.0.0 and add a note about it in the readme to say that it is partially maintained and use at your own risk...

**Comment by atarix83** [comment]
any update to this? it would be nice if merged

**Comment by rafa-as** [comment]
A new PR will be merged with v13 compatibility. Thanks!

**Pull request closed without merging**